tests/unit/components/article-wrapper-test.js

```diff
@@ -112,26 +101,3 @@ test('wiki with disableAnonymousEditing set requires log in', function (assert)
 		expected: false
 	}, 'editAllowed', assert);
 });
-
-test('featured video visible for articles without infoboxes and featured video set', function (assert) {
```

Comment:
don't we want to leave a test checking if video is visible for articles with featured video set? or it's too obvious?